### DISCUSSION TITLE
Customize login view

### DISCUSSION THREAD

**Discussion by juljoshahini** [firecmsco/firecms#477] [Q&A] [answered]
I am trying to customize the login view, add a background image, a container etc. I am having some trouble to find how to customize the login view. I want to add these but not affect the other pages. I am having some trouble finding any info regarding custom styling

**Comment by fgatti675** [comment, marked as answer]
Hi @juljoshahini 
If you need a large degree of customisation for your login view, I suggest going down the custom CMS route:
https://firecms.co/docs/custom_cms_app
That way you can even replace the login view completely if you need it.
The source code for the current login view is here:
https://github.com/Camberi/firecms/blob/main/lib/src/firebase_app/components/FirebaseLoginView.tsx
You can build your own component using it as a reference :)
